angular/projects/admin-nrpti/src/app/services/document.service.ts

```diff
@@ -13,7 +13,7 @@ import { HttpClient } from '@angular/common/http';
  */
 @Injectable({ providedIn: 'root' })
 export class DocumentService {
-  constructor(public apiService: ApiService, public http: HttpClient) {}
+  constructor(public apiService: ApiService, public http: HttpClient) { }
```

Comment:
Same as above, nothing here has really changed, except for a space between the brackets at the end.